components/classroom/main.js

```diff
@@ -47,7 +55,18 @@ window.onload = () => {
     }
 };
 
-const ROLE_HIERARCHY = [Roles.OWNER, Roles.LEADER, Roles.CONTRIBUTOR];
+PROJECT_FORM.addEventListener('submit', async function (event) {
+    event.preventDefault();
+    const projectID = document.getElementById('project-id').value;
+    location.href = '?projectID=' + projectID;  // Redirect to the same page with the projectID in the query string
+});
+
+clickList.addEventListener('click', function (event) {
+    const LI = event.target.closest('LI');
+    if (LI) {
+        location.href = '?projectID=' + event.target.getAttribute('tpen-project-id');
+    }
+});
```

Comment:
lines 58-69 will be inside the fetchProjectData function--we still have to decide which file we want fetchProjectData in so we reduce code redundancy